apps/wallet/src/ui/app/components/user-approve-container/index.tsx

```diff
@@ -55,37 +54,33 @@ export function UserApproveContainer({
 	const parsedOrigin = useMemo(() => new URL(origin), [origin]);
 
 	return (
-		<div className={st.container}>
-			<div className={cl(st.scrollBody, { [st.scrollable]: scrollable })}>
+		<div className="flex flex-1 flex-col flex-nowrap h-full">
+			<div className="flex-1 pb-0 flex flex-col">
 				<DAppInfoCard
 					name={parsedOrigin.host}
 					url={origin}
+					permissions={permissions}
 					iconUrl={originFavIcon}
 					connectedAddress={!addressHidden && address ? address : undefined}
 				/>
-				<div className={cl(st.children, { [st.scrollable]: scrollable, [st.blended]: blended })}>
-					{children}
-				</div>
+				<div className="flex flex-1 flex-col p-6 bg-hero-darkest/5">{children}</div>
 			</div>
-			<div className={st.actionsContainer}>
+			<div className="sticky bottom-0 ">
 				<div
-					className={cl(st.actions, isWarning && st.flipActions, {
-						[st.blended]: blended,
-						[st.blurBorder]: blended,
+					className={cn('bg-hero-darkest/5 backdrop-blur-lg py-4 px-5 flex items-center gap-2.5', {
+						'flex-row-reverse': isWarning,
 					})}
 				>
 					<Button
 						size="tall"
-						variant="warning"
+						variant="secondary"
 						onClick={() => {
 							handleOnResponse(false);
 						}}
 						disabled={submitting}
 						text={rejectTitle}
 					/>
 					<Button
-						// recreate the button when changing the variant to avoid animating to the new styles
```

Comment:
The button looked weird when it was transitioning from secondary to primary. Did you remove this intentionally?

Reply:
this was only relevant when we had a blur effect that was removed with the UI updates